tickets/src/__tests__/middlewares/eventMiddleware.test.js

```diff
@@ -46,6 +48,48 @@ describe('Event middleware', () => {
     mockeventService = {}
   })
 
+  describe('handling ADD_EVENT', () => {
+    it('should save an event from the provided object', async () => {
+      expect.assertions(2)
+      const { next, invoke, store } = create()
+
+      const eventDate = new Date()
+
+      const action = {
+        type: ADD_EVENT,
+        event: {
+          name: 'Arbitrary name',
+          date: eventDate,
+          lockAddress: '0x123',
+          description: 'Arbitrary description',
+          location: 'My house',
+          duration: 360,
+          logo: 'Some string',
+          image: 'Image hash',
+          outOfSchemaField: 'What is this doing here?!',
+        },
+      }
+
+      const data = UnlockEvent.build({
+        name: 'Arbitrary name',
+        date: eventDate,
+        lockAddress: '0x123',
+        description: 'Arbitrary description',
+        location: 'My house',
+        duration: 360,
+        logo: 'Some string',
+        image: 'Image hash',
```

Comment:
I assumed that image would reference the URI of the image is this what you pictured as well, or were you thinking about the blob of the image?

Reply:
Yeah, you're totally right. I was actually thinking filename, but URI makes much more sense. I'll adjust the string here for clarity.